### PR TITLE
chore: ensure we can connect to objecten/objecttypen when running ZAC from IntelliJ

### PR DESCRIPTION
**Pull request by felixcicatt**
To enable testing the OpenFormulieren flow when running ZAC from IntelliJ, add variables for the object / object type registrations to env.tpl, make the OpenZaak / OpenObjecten configuration compatible with that scenario, and use BSNs from the `Testdataset persoonslijsten proefomgevingen gba-v` in the demo data we use for the Productaanvraag.
Resolves PZ-2724

### CODE REVIEW

File: .env.tpl [hunk 1]
```diff
@@ -56,6 +56,10 @@ SMTP_SERVER=op://Dimpact/ZAC-.env-$APP_ENV/SMTP/SERVER
 SMTP_PORT=op://Dimpact/ZAC-.env-$APP_ENV/SMTP/PORT
 SMTP_USERNAME=op://Dimpact/ZAC-.env-$APP_ENV/MAILJET/API_KEY
 SMTP_PASSWORD=op://Dimpact/ZAC-.env-$APP_ENV/MAILJET/API_SECRET_KEY
+OBJECTS_API_CLIENT_MP_REST_URL=http://host.docker.internal:8011
+OBJECTS_API_TOKEN=182c13e2209161852c53cef53a879f7a2f923430
```
Comment (edgarvonk) [inline .env.tpl:60]: We can probably also replace this with e.g. `dummyZacObjectsToken` right? If of course we also change it the `demodata.json` source file.

File: scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql [hunk 2]
```diff
@@ -10,4 +10,4 @@ INSERT INTO documenten_enkelvoudiginformatieobject (id, identificatie, bronorgan
...
 -- Set up the Objecten service configuration.
-INSERT INTO zgw_consumers_service (label, api_type, api_root, client_id, secret, auth_type, header_key, header_value, oas, nlx, user_id, user_representation, oas_file, client_certificate_id, server_certificate_id, uuid) VALUES ('Objects API', 'orc', 'http://objecten-api.local:8000/api/v2/', '', '', 'api_key', 'Authorization', 'Token cd63e158f3aca276ef284e3033d020a22899c728', 'http://objecten-api.local:8000/api/v2/schema/openapi.yaml', '', '', '', '', NULL, NULL, '9fe5f3fb-a1f5-4ea5-a2fe-c5b1294da0f4');
+INSERT INTO zgw_consumers_service (label, api_type, api_root, client_id, secret, auth_type, header_key, header_value, oas, nlx, user_id, user_representation, oas_file, client_certificate_id, server_certificate_id, uuid) VALUES ('Objects API', 'orc', 'http://host.docker.internal:8010/api/v2/', '', '', 'api_key', 'Authorization', 'Token cd63e158f3aca276ef284e3033d020a22899c728', 'http://host.docker.internal:8010/api/v2/schema/openapi.yaml', '', '', '', '', NULL, NULL, '9fe5f3fb-a1f5-4ea5-a2fe-c5b1294da0f4');
```
Comment (edgarvonk) [inline scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql:13]: Hmm. I'm surprised this even works because `host.docker.internal` definitely is not reachable from Docker Compose when running our integration tests in GitHub. That is the reason why we don't use `host.docker.internal` in any of there startup scripts. Really the use of `host.docker.internal` should be restricted where we really needed and not here I think. It is a workaround for running things locally really.

File: scripts/docker-compose/imports/objects-api/fixtures/demodata.json [hunk 3]
```diff
@@ -840,13 +840,13 @@
                 },
                 "betrokkenen" : [
                     {
-                        "inpBsn" : "111222333",
+                        "inpBsn" : "999993896",
```
Comment (edgarvonk) [inline scripts/docker-compose/imports/objects-api/fixtures/demodata.json:843]: Why was this change needed?

**Review by edgarvonk — CHANGES_REQUESTED** [review]

File: scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql (hunk 2 above)
Comment (edgarvonk) [reply scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql:13]: Or maybe `host.docker.internal` is reachable by GitHub with @hsiliev 's Linux trick now..? I dunno.. But even then in these scripts we should not use it I think. I see no need for it. Just stick with the internal Docker container names.

Comment (felixcicatt) [reply scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql:13]: That's probably why the tests are failing now. Should we then add a developer instruction to map this via their host file? So `objecten-api.local` -> `localhost:8010`?

File: scripts/docker-compose/imports/objects-api/fixtures/demodata.json (hunk 3 above)
Comment (felixcicatt) [reply scripts/docker-compose/imports/objects-api/fixtures/demodata.json:843]: These values map to test data from the GBA. If you send a 'mock' Notification to the ZAC endpoint for it, you end up with a new Zaak with an Initiator that's actually found in the BRP. I could for the life of me not get a new object into the objects registration from the admin UI so this was the easiest way in the end.

File: .env.tpl (hunk 1 above)
Comment (felixcicatt) [reply .env.tpl:60]: Good point!

File: scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql (hunk 2 above)
Comment (edgarvonk) [reply scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql]: Not sure. This is pretty standard Docker Compose port mapping stuff right? The same for all other mapped ports we have.

File: scripts/docker-compose/imports/objects-api/fixtures/demodata.json (hunk 3 above)
Comment (edgarvonk) [reply scripts/docker-compose/imports/objects-api/fixtures/demodata.json:843]: Ah yes, good old Héndrika Janse.. :-) Clear.

**Review by edgarvonk — APPROVED** [review]

File: scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql (hunk 2 above)
Comment (felixcicatt) [reply scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql]: The thing is, when we run ZAC from IntelliJ, it's not part of the docker compose setup, so it can't reach the containers by host name. I'll see if I can get it to work by changing my host file.

Comment (edgarvonk) [reply scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql]: ZAC is not but Objecten is still part of the Docker Compose setup in that situation.

Comment (felixcicatt) [reply scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql]: I'm not getting my point across, let's have a call about this later.
